### DISCUSSION TITLE
make_trade ends with error but no material exception provided

### DISCUSSION THREAD

**Discussion by wkaszycki** [uniswap-python/uniswap-python#395] [General]
Hello guys! I am trying this library, and it seems like the code to make_trade does not work properly and does not produces any readable exceptions. Did anyone experience similar situation?

            uniswap = Uniswap(address=sub_wallet.address, private_key=sub_wallet._private_key.hex(), version=2,
                              provider=INFURA_URL)
            try:

                tx = uniswap.make_trade(
                    amount_in,  # ETH to spend
                    token_address,  # token to buy
                    {'from': sub_wallet.address, 'gas': trade_gas_limit, 'gasPrice': gas_price}
                )
                receipt = w3.eth.wait_for_transaction_receipt(tx)

            except Exception as e:
                results.append(f"Error on wallet {sub_wallet.address}: {str(e)}")

**Comment by hoalongdao** [comment]
import logging
logging.basicConfig(level=logging.DEBUG)
